debian/control

```diff
@@ -0,0 +1,18 @@
+Source: nasp-kernel-module
+Section: kernel
+Priority: optional
+Maintainer: Nandor Kracser <nandork@cisco.com>
+Rules-Requires-Root: no
+Build-Depends:
+ debhelper-compat (= 13),
+Standards-Version: 4.6.2
+Homepage: https://github.com/cisco-open/nasp-kernel-module
+Vcs-Browser: https://github.com/cisco-open/nasp-kernel-module
+Vcs-Git: https://github.com/cisco-open/nasp-kernel-module.git
+
+Package: nasp-kernel-module
+Architecture: all
```

Comment:
I think this will be wrong since this package is not architecture-independent.
 https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture
 I think we need to specify the supported archs like arm64 amd64

Reply:
Since we are building this on the users machine this will be platform independent.